src/main/java/com/jcabi/dynamodb/maven/plugin/CreateTablesMojo.java

```diff
@@ -66,6 +67,7 @@
  * @version $Id$
  * @checkstyle ClassDataAbstractionCoupling (300 lines)
  * @checkstyle MultipleStringLiterals (300 lines)
+ * @checkstyle ExcessiveImports (400 lines)
```

Comment:
@piddubnyi `ExcessiveImports` is PMD rule, checkstyle exclude will not work with it